shared/chat/conversation/list/container.js

```diff
@@ -105,6 +106,7 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
     dispatch(navigateAppend([{props: {message}, selected: 'messageAction'}]))
   },
   onOpenInFileUI: (path: string) => dispatch(({payload: {path}, type: 'fs:openInFileUI'}: OpenInFileUI)),
+  onUsernameClick: (username: string) => dispatch(onUserClick(username)),
```

Comment:
I'd like to actually have this bring the tracker popup up, as described in the ticket, as otherwise the back button on the profile page doesn't take you back to chat.

I had some trouble finding the right incantation for that, but I'll look tomorrow...